plugins/router/f5/f5.go

```diff
@@ -723,7 +841,7 @@ func (f5 *f5LTM) CreatePool(poolname string) error {
 
 	payload := f5Pool{
 		Mode:    "round-robin",
-		Monitor: "/Common/http",
+		Monitor: fmt.Sprintf("%s/http", f5.partitionPath),
```

Comment:
No idea really - don't know too much about F5 to comment on that. 
I'll switch it back to ```/Common/http``` and add a comment above this.